metrics/stats.go

```diff
@@ -40,8 +40,13 @@ type Stats struct {
 	GcPerSecond      float64   `json:"gc_per_second"`
 	GcPausePerSecond float64   `json:"gc_pause_per_second"`
 	GcPause          []float64 `json:"gc_pause"`
-	// Connections
-	Connections int64 `json:"connections"`
+}
+
+type PlasmaStats struct {
```

Comment:
__[golint]__
* [golint] exported type PlasmaStats should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->